Source/bmalloc/libpas/src/libpas/pas_deallocate.c

```diff
@@ -120,7 +148,12 @@ bool pas_try_deallocate_slow_no_cache(void* ptr,
     }
     
     begin = (uintptr_t)ptr;
-        
+
+    /* Try to deallocate a PGM allocation based on config and checking PGM entry */
+    bool pgm_return = pas_try_deallocate_pgm_large(ptr, config_ptr);
+    if (pgm_return)
+        return true;
```

Comment:
The more idiomatic way to write this in WebKit is:

```suggestion
    if (pas_try_deallocate_pgm_large(ptr, config_ptr))
        return true;
```

Reply:
Fixed